content/challenges/sf-pi-car-v.en.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "SF-PICar-V Computer Vision with SimpleCV"
+levels:
+    - Intermediate
+    - Advanced
+hardware:
+    - SF-PiCar-V
+    - Sensors of your choice
```

Comment:
Should be  "_Sensores of choice_" like in every other challenge.

Reply:
I will change every english challenge to "Sensors of your choice" as it is more directed towards potential attendees.